tesseract/tesseract_planning/tesseract_motion_planners/include/tesseract_motion_planners/trajopt/config/trajopt_planner_default_config.h

```diff
@@ -135,6 +135,19 @@ struct TrajOptPlannerDefaultConfig : public TrajOptPlannerConfig
    */
   std::vector<std::tuple<sco::VectorOfVector::func, sco::MatrixOfVector::func, sco::ConstraintType, Eigen::VectorXd>>
       constraint_error_functions;
+
+protected:
+  bool checkUserInput() const;
+  bool addBasicInfo(trajopt::ProblemConstructionInfo& pci) const;
+  bool addInitTrajectory(trajopt::ProblemConstructionInfo& pci) const;
+  void addWaypoints(trajopt::ProblemConstructionInfo& pci, std::vector<int>& fixed_steps) const;
```

Comment:
`void addWaypoints(trajopt::ProblemConstructionInfo& pci, std::vector<int>& fixed_steps = std::vector<int>())` ?

Reply:
I assume you are referring the fixed_steps parameter. It should be removed.

Reply:
Sorry I was incorrect. This is required. The fixed_steps are defined by the type of waypoints. If provided a waypoint of type JointWaypoint if would be set as a fixed_step all other types at the moment are considered not fixed steps.